_rules/meta-viewport-b4f0c3.md

```diff
@@ -22,11 +22,11 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport`.
+This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport`. In addition, the `content` [attribute value][] has the `user-scalable` or `maximum-scale` properties.
```

Comment:
This is better aligned with other rules and eliminates any eventual ambiguity when the attribute includes the two properties (in case the "or" is not understood as a "logical or").

```suggestion
This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
- the `content` [attribute value][] has the `user-scalable` property; or
- the `content` [attribute value][] has the `maximum-scale` property.
```